cohortextractor/patients.py

```diff
@@ -31,6 +59,29 @@ def age_as_of(
     # Required keyword
     return_expectations=None,
 ):
+    """
+    Returns age of patient of at a particular date
```

Comment:
Note that age can be negative if a patient was born after the reference date!

Reply:
thanks - i have corrected